src/core/metrics/app-metrics.ts

```diff
@@ -1,5 +1,11 @@
+import { defer, every } from 'lodash';
```

Comment:
i am not see where defer used. Also - i highly recommend  import lodash functions separately 
import  defer from 'lodash/defer'

https://lodash.com/per-method-packages;